rad-profile/src/cli/args.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright © 2021 The Radicle Link Contributors
+//
+// This file is part of radicle-link, distributed under the GPLv3 with Radicle
+// Linking Exception. For full terms see the included LICENSE file.
+
+use structopt::StructOpt;
+
+use librad::profile::ProfileId;
+
+/// Management of Radicle profiles and their associated configuration data.
+#[derive(Debug, StructOpt)]
+pub struct Args {
+    #[structopt(subcommand)]
+    pub command: Command,
+}
+
+#[derive(Debug, StructOpt)]
+pub enum Command {
+    Create(Create),
+    Get(Get),
+    Set(Set),
+    List(List),
+    Peer(GetPeerId),
+    Paths(GetPaths),
+    SshAdd(SshAdd),
+}
+
+/// Create a new profile, generating a new secret key and initialising
+/// configurations and storage.
+#[derive(Debug, StructOpt)]
+pub struct Create {}
+
+/// Get the currently active profile.
+#[derive(Debug, StructOpt)]
+pub struct Get {}
```

Comment:
Could this take an optional `id` to get another profile than the currently active?

Reply:
If not it might be more accurately named `GetActive`.

Reply:
Ya, that's a good point. Need to revisit why I didn't do that initially :)